<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignalSlot</class>
 <widget class="QMainWindow" name="SignalSlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignalSlot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_Slider_Progressbar">
     <item>
      <widget class="QSlider" name="horizontalSlider_signalslot">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar_signalslot">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_enable_signalslot">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Signal Slot&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disable_signalslot">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable Signal Slot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_exit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Close (Exit)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSignal">
    <property name="title">
     <string>Signal</string>
    </property>
   </widget>
   <addaction name="menuSignal"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close_exit</sender>
   <signal>clicked()</signal>
   <receiver>SignalSlot</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
